tracer/src/Datadog.Trace/SignalFx/Metrics/SignalFxReporter.cs

```diff
@@ -69,7 +69,7 @@ public void Send(DataPointUploadMessage msg)
             }
             catch (Exception ex)
             {
-                Log.Error($"Exception sending metrics: {ex.Message}");
+                Log.Error(ex, "An error occurred while sending metrics.");
```

Comment:
```suggestion
                Log.Error(ex, "Metric data could not be sent.");
```

Can we make this error more informative? What does `{ex.Message}` contains? Can we add perhaps the endpoint URL in the error?